src/game/cheats.c

```diff
@@ -165,8 +165,19 @@ void cheatActivate(s32 cheat_id)
 		setCurrentPlayerNum(prevplayernum);
 		break;
 	case CHEAT_ALLGUNS:
+#ifndef PLATFORM_N64
```

Comment:
Do we want to make this work in Combat Simulator? I think set, let the user decide. But we could easily turn off that part of the cheat by adding back `if (g_Vars.normmplayerisrunning == false) {`